config/initializers/100-silence_logger.rb

```diff
@@ -20,9 +20,9 @@ def call(env)
     if env[HTTP_X_SILENCE_LOGGER] || @opts[:silenced].include?(path_info) ||
          path_info.start_with?("/logs") || path_info.start_with?("/user_avatar") ||
          path_info.start_with?("/letter_avatar")
-      if ::Logster::Logger === Rails.logger
+      if defined?(::Logster::Logger) && Logster.logger
         override = true
-        Rails.logger.override_level = Logger::WARN
+        Logster.logger.override_level = Logger::WARN
```

Comment:
This is resulting in spec failures following the force push @Flink

Reply:
Yes I just updated again, they should be fixed :grin: